lua/toggleterm/config.lua

```diff
@@ -17,10 +20,34 @@ local config = {
   shell = vim.o.shell,
   float_opts = {
     winblend = 0,
-    highlights = {
-      background = "Normal",
-      border = "Normal",
-    },
+  },
+}
+
+config.highlights = {
+  Normal = {
+    guibg = colors.shade_color(colors.get_hex("Normal", "bg"), constants.shading_amount),
+  },
+  NormalFloat = {
```

Comment:
@aspeddro can these be `snake_case` rather than PascalCase, not sure why they are if it's because of converting the names, so they can be used in a map then you can use
```lua
  -- convert 'highlight_value' to 'HighlightValue' -> snake to pascal
  local formatted = name:gsub("_(.)", name.upper):gsub("^%l", string.upper)
```
I think this would look much tidier if the casing was consistent

Reply:
@akinsho keeping PascalCase makes mapping easier. One problem with the conversion is that it doesn't work for `StatusLineNC`, you would have to use `status_line_n_c`.

Another facility, passing the arguments to `vim.highlights.create` is made simpler.